Bio/SeqFeature.py

```diff
@@ -1760,6 +1795,7 @@ class AfterPosition(int, AbstractPosition):
 
     # Subclasses int so can't use __init__
     def __new__(cls, position, extension=0):
+        """Create a new instance in AfterPosition object."""
```

Comment:
Odd phrasing, how about: ``Create a new instance of the AfterPosition object.`` here?